lib/connectionConfig/connectionConfig.ts

```diff
@@ -80,6 +80,11 @@ export interface ConnectionConfig {
    * connection over WebSockets.
    */
   webSocketEndpointPath?: string;
+  /***
+     * @property {any} {socketOptions} - Options to be passed to the function returned by
+     * rhea.websocket_connect()
+     */
+  socketOptions?: any;
```

Comment:
Do we want to track the work for making this typed based on the constructor passed in as the webSocket configuration key?

Reply:
Yes, please create an issue and link it to the epic https://github.com/Azure/azure-sdk-for-js/issues/1056

Reply:
Once we will get response back from the user, I'll create a separate PR for that.

Reply:
@ramya-rao-a Yes, I'll create an issue.